Description: Fix FTBFS for binutils-gold
 Add the missing -lengine to fix FTBFS.
Author: Ying-Chun Liu (PaulLiu) <paulliu@debian.org>
Bug-Debian: http://bugs.debian.org/615701
Last-Update: 2011-03-15
Index: clutter-gesture-0.0.2.1/tests/Makefile.am
===================================================================
--- clutter-gesture-0.0.2.1.orig/tests/Makefile.am	2011-03-15 20:17:41.772365741 +0800
+++ clutter-gesture-0.0.2.1/tests/Makefile.am	2011-03-15 21:14:21.952365914 +0800
@@ -29,7 +29,7 @@
 
 test_engine_SOURCES = test_engine.c
 test_engine_CFLAGS = @CLUTTERGESTURE_CFLAGS@ -g3 -O0 -ggdb -DPKGDATADIR="\"$(pkgdatadir)\""
-test_engine_LDFLAGS = $(top_builddir)/clutter-gesture/libcluttergesture-@PACKAGE_VERSION@.la
+test_engine_LDFLAGS = $(top_builddir)/clutter-gesture/libcluttergesture-@PACKAGE_VERSION@.la -L../engine/.libs -lengine
 
 CLEANFILES = *~ test-events, test_engine
 
